torch/onnx/symbolic_opset10.py

```diff
@@ -145,12 +146,12 @@ def _avg_pool(name, tuple_fn):
     @symbolic_helper.parse_args("v", "is", "is", "is", "i", "i", "none")
     def symbolic_fn(
         g,
-        input,
-        kernel_size,
-        stride,
-        padding,
-        ceil_mode,
-        count_include_pad,
+        input: _C.Value,
+        kernel_size: Tuple[int, ...],
```

Comment:
parsed by "is", so `List` would be more accurate

Reply:
Changed to Sequence which can accomodate the List output from parse_args